specifyweb/frontend/js_src/lib/tests/jsdom.js

```diff
@@ -1,18 +1,17 @@
 import { JSDOM } from 'jsdom';
 
 // Create a new JSDOM instance
+/**
+ * FEATURE: Allow customizing the JSDOM userAgent, platform, and other global
+ * properties
+ * See https://github.com/jsdom/jsdom#advanced-configuration
+ * See https://github.com/specify/specify7/pull/5389
```

Comment:
that should probably be done using mocking:

in beforeAll in some test file set userAgent/platform/...
in afterAll undo the change